docs/A-Introduction/compilers.md

```diff
@@ -205,11 +207,11 @@ int main(void)           // program startup
 }
 ```
 
-When we or the users load the executable code into RAM \(a.out or hello.exe\), the operating system transfers control to this entry point. The last statement \(return 0;\) before the closing brace transfers control back to the operating system.
+When the users or we load the executable code into RAM \(`a.out` or `hello.exe`\), the operating system transfers control to this entry point. The last statement \(`return 0;`\) before the closing brace transfers control back to the operating system.
 
 ### Program Output
 
-The following statement outputs "This is C" to the standard output device \(for example, the screen\).
+The following statement outputs `"This is C"` to the standard output device \(for example, the screen\).
```

Comment:
This line and line 85 are similar in my opinion, so I think you don't have to wrap this in backticks

Reply:
Thank you, I will fix it